bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/config/config.xml

```diff
@@ -103,6 +103,18 @@
 		</parameter>
 	</config-description>
 
+	<config-description uri="netatmo:camera">
```

Comment:
No need to change thing configuration to point to this new configuration ?

Reply:
It is needed because I introduce a complementary parameter for Camera's (ipAddress). So it is needed to have a distinct config description. But you're pointing me to the fact that I'll have to take care of migration insctructions.

Reply:
I understood why you defined a new configuration. My question was rather if it is really used, I mean pointed by any thing type !

Reply:
Yes, for Welcome, Doorbell and Presence, it is the modification done in ModuleType.java

Reply:
> Yes, for Welcome, Doorbell and Presence, it is the modification done in ModuleType.java
Ok

Reply:
@lolodomo : I'm afraid instruction to change a thing-type definition does not exist in update instructions.
I've see two options 
- adding "localIP" configuration element to all things. It will be used only for these devices as of today.
- notice about the  need to recreate Camera, Doorbell, Presence things (not mandatory but nice to avoid this bug).

Reply:
I can change the coding so having deprecated configuration (without localIp) will not prevent the binding from working, only they will have a warning in log saying that a suspicious local address is detected.

Reply:
Normally enhancing a thing configuration does not require upgrade instructions, the new configuration is automatically considered. But in your case you change configuration URI.
You should test what happens, maybe that is simply ok.